# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vlc/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- vlc-0.8.4a/modules/gui/skins2/utils/var_tree.hpp_orig	2006-03-29 23:22:07.526917000 +0200
+++ vlc-0.8.4a/modules/gui/skins2/utils/var_tree.hpp	2006-03-29 23:22:28.332217250 +0200
@@ -88,7 +88,7 @@
 
         /// Parent node
         VarTree *parent() { return m_pParent; }
-        void VarTree::checkParents( VarTree *pParent );
+        void checkParents( VarTree *pParent );
 
         Iterator uncle();
 
--- vlc-0.8.4a/modules/gui/skins2/x11/x11_graphics.hpp_orig	2006-03-29 23:30:50.555604250 +0200
+++ vlc-0.8.4a/modules/gui/skins2/x11/x11_graphics.hpp	2006-03-29 23:31:17.433284000 +0200
@@ -98,10 +98,10 @@
         GC m_gc;
 
         /// Add an horizontal segment in a region
-        void X11Graphics::addHSegmentInRegion( Region &rMask, int xStart,
+        void addHSegmentInRegion( Region &rMask, int xStart,
                                                int xEnd, int y );
         /// Add a vertical segment in a region
-        void X11Graphics::addVSegmentInRegion( Region &rMask, int yStart,
+        void addVSegmentInRegion( Region &rMask, int yStart,
                                                int yEnd, int x );
 };
 
